### DISCUSSION TITLE
小唐卡蜜拉EGO这个被动的翻译是不是有点问题

### DISCUSSION THREAD

**Discussion by 520030910343** [LocalizeLimbusCompany/LocalizeLimbusCompany#298] [问答 | Q&A]
“Every time 10” 个人觉得应该翻译成“每次10”而不是“每10次”比较好吧（而且总不能吃满要一回合触发100次流血吧，那也太幽默了）

**Comment by Nephthys137** [comment]
@mucketguct

**Comment by mucketguct** [comment]
您好，感谢反馈，这里确实是一处笔误，现已修正啦，会在下次模组更新中同步此修正，感谢反馈，也祝玩得开心啦
